R/make_choose_all_table.R

```diff
@@ -36,29 +36,33 @@ getLabel2 <- function(data, aVariable) {
 #' @return A variable's response label without  the choose all the question
 #'
 ## @examples
-##\dontrun{
+## \dontrun{
 ##  make_choose_all_table(redcap, "ingredients")
 ##  }
 make_choose_all_table <- function(df, variable) {
-  
   # . <- NULL # kludge to get CMD Check to pass with nonstandard evaluation
   counts <- df %>%
     dplyr::select(dplyr::starts_with(variable)) %>%
     dplyr::mutate(across(everything(), ~ . %in% c("1", "Checked"))) %>%
```

Comment:
add @importFrom tidyselect everything starts_with
add tidyselect to package dependencies